src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToStartEventHandler.java

```diff
@@ -130,4 +132,15 @@ private boolean isDwpResponseDocsMissing(SscsCaseData sscsCaseData) {
     private boolean hasMandatoryDocumentMissingForLegacyAppeals(List<DwpDocument> dwpResponseDocs) {
         return dwpResponseDocs.stream().anyMatch(e -> isNull(e.getValue().getDocumentLink()));
     }
+
+    private void moveDocsToDwpCollectionIfOldPattern(SscsCaseData sscsCaseData) {
+        //Before we moved to the new DWP document collection, we stored DWP documents within their own fields. This would break bundling with the new config that
+        //looks at the new DWP document collection. Therefore, if the DWP fields are populated, then assume old pattern and move to the DWP document collection.
```

Comment:
I've read this 3 times, I don't understand it. Some terms I'm unfamiliar with are `new config` and `old pattern`

Reply:
Its Jacks change from the old pattern of individual DWP documents to adding them to the dwpDocuments collection, rather than do a migration its been coded into update whenever we deal with dwp documents. It should get sorted in dwpUploadResponse but we have legacy appeals in Prod that haven't been updated.